src/routes/connect/connectHelpers.ts

```diff
@@ -0,0 +1,19 @@
+import { Keypair } from 'stellar-sdk';
+import { publicKey } from '../../stores/store';
+import { connectionError } from './connectStores';
```

Comment:
the helper should not access the store, it should only know how to get the private key or throw an error. The store should be accessed by the component.